lib/workers/repository/config-migration/branch/migrated-data.ts

```diff
@@ -81,10 +92,19 @@ export class MigratedDataFactory {
     return this.data;
   }
 
-  public static reset(): void {
+  static reset(): void {
     this.data = null;
   }
 
+  static applyPrettierFormatting({
+    content,
+    filename,
+    indent,
+  }: MigratedData): Promise<string> {
+    const parser = upath.extname(filename).replace('.', '') as PrettierParser;
+    return applyPrettierFormatting(content, parser, indent);
+  }
```

Comment:
this should probably be moved to lib/utils in a later PR, as it's not directly related to migration.

Reply:
no problem ill move it later on